src/batch-client/index.ts

```diff
@@ -175,26 +175,20 @@ export class ZimbraBatchClient {
 	public accountInfo = () =>
 		this.jsonRequest({
 			name: 'GetInfo',
-			namespace: Namespace.Account
-		}).then(res => {
-			let prefs: any = mapValuesDeep(res.prefs._attrs, coerceStringToBoolean);
-			prefs.zimbraPrefMailTrustedSenderList =
-				typeof prefs.zimbraPrefMailTrustedSenderList === 'string'
-					? castArray(prefs.zimbraPrefMailTrustedSenderList)
-					: prefs.zimbraPrefMailTrustedSenderList;
-			return {
-				...res,
-				attrs: mapValuesDeep(res.attrs._attrs, coerceStringToBoolean),
-				prefs,
-				identities: mapValuesDeep(res.identities, coerceStringToBoolean),
-				...(get(res, 'license.attr') && {
-					license: {
-						status: res.license.status,
-						attr: mapValuesDeep(res.license.attr, coerceStringToBoolean)
-					}
-				})
-			};
-		});
+			namespace: Namespace.Account,
+			body: {
+				sections: 'mbox,attrs,zimlets'
```

Comment:
FYI - This is to define the kind of data expected from response of GetInfoRequest.